crates/ruff/src/rules/pyflakes/rules/f_string_missing_placeholders.rs

```diff
@@ -47,31 +47,49 @@ impl AlwaysAutofixableViolation for FStringMissingPlaceholders {
     }
 }
 
-/// Find f-strings that don't contain any formatted values in an [`FString`].
+/// Return an iterator containing a two-element tuple for each f-string in the
+/// given [`ExprFString`] expression. The first element of the tuple is the
+/// f-string prefix range, and the second element is the entire f-string range.
+///
+/// For example,
+///
+/// ```python
+/// f"example"
+/// ```
+///
+/// would return `[(0..1, 0..10)]`.
+///
+/// This function assumes that the given expression is an [`ExprFString`]
+/// without any placeholder expressions.
+///
+/// [`ExprFString`]: `ruff_python_ast::ExprFString`
 fn find_useless_f_strings<'a>(
     expr: &'a Expr,
     locator: &'a Locator,
     source_type: PySourceType,
 ) -> impl Iterator<Item = (TextRange, TextRange)> + 'a {
     let contents = locator.slice(expr);
+    let mut current_f_string_start = TextSize::default();
```

Comment:
Nit: Just to make sure we never index from the start of the document

```suggestion
    let mut current_f_string_start = expr.start();
```